### ISSUE TITLE
Diff fails on Windows

### ISSUE THREAD

**Issue by colinking** [JoshuaKGoldberg/eslint-plugin-expect-type#63]
### 🐛 Bug Report

- `eslint-plugin-expect-type` version: 0.2.1
- ESLint version: 8.26.0
- Node version: 18

#### Actual Behavior

The diff produced on mac/linux is correct, but on Windows the expected diff includes an extraneous line which causes eslint to error:

```
  13:10  error  Expected type to be: type _response = {
    id: string;
    taskID?: string | undefined;
    paramValues: ParamValues;
    status: RunStatus;
    output: string;
}
, got: type _response = {
    id: string;
    taskID?: string | undefined;
    paramValues: ParamValues;
    status: RunStatus;
    output: string;
}  expect-type/expect
```

#### Expected Behavior

The diff should match.

#### Reproduction

Here's an example CI run that shows the error: https://github.com/airplanedev/js-sdk/actions/runs/3440332462/jobs/5738676594

We ended up disabling this on Windows as a workaround.

**Comment by JoshuaKGoldberg** [comment]
Thanks for the report - and again apologies for the delay! Accepting PRs to fix this Windows-specific bug.

I'm out traveling now but have a Windows laptop at home that can I can use later this week. If nobody's sent a PR by then I'll aim to. Hopefully. 🤞

**Comment by JoshuaKGoldberg** [comment]
Interestingly, it looks like it's _both_ trimming and `\r\n` vs `\n`:

```js
{
  expected: 'const vector: {\n    x: number;\r\n    y: number;\r\n}\r\n',
  actual: 'const vector: {\n    x: number;\n    y: number;\n}'
}
```

**Comment by JoshuaKGoldberg** [comment]
This should be available in [v0.2.2](https://www.npmjs.com/package/eslint-plugin-expect-type/v/0.2.2). Cheers!
